sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/agents/implementation/models/ErrorAdditionalInfo.java

```diff
@@ -21,16 +22,19 @@ public final class ErrorAdditionalInfo implements JsonSerializable<ErrorAddition
     /*
      * The additional info type.
      */
+    @Generated
```

Comment:
@alzimmermsft Is there a particular reason that the apis now have this `@generated` annotation? I didn't see a change in the swagger readme. Is this from the autorest version upgrade?

Reply:
This was added in one of the versions of Autorest between 4.1.42 and the upgrade to 4.1.50 here.